Examples/test-suite/common.mk

```diff
@@ -700,6 +705,7 @@ all: $(NOT_BROKEN_TEST_CASES) $(BROKEN_TEST_CASES)
 broken: $(BROKEN_TEST_CASES)
 
 check: $(NOT_BROKEN_TEST_CASES)
+	@echo $(shell echo $^|wc -w) tests passed
```

Comment:
I somehow completely forgot about `$(words)` :-( The worst is that I was sure that there was a function to do this in GNU make and spent some time looking for it, but failed to find it. I'll update this, of course, thanks!

As for redirecting make output, I also do this, of course, but I prefer to just be able do `make check-xxx-test-suite` and check there is no error at the end, this is much less error-prone.